packages/rocketchat-language-plugin/server/server.js

```diff
@@ -0,0 +1,46 @@
+Meteor.methods({
+	getroom() {
+		const user_id = this.userId;
+		const priority_language_fullname = Meteor.call('get_language');
+		const user_info = RocketChat.models.Users.findOneById(user_id);
```

Comment:
What if user is not logged in when this method is called? This will return null.